openslides_backend/action/motion_category/create_update_delete.py

```diff
@@ -0,0 +1,20 @@
+from ...models.motion_category import MotionCategory
+from ..action import register_action_set
+from ..action_set import ActionSet
+from ..default_schema import DefaultSchema
+
+
+@register_action_set("motion_category")
+class MotionCategoryActionSet(ActionSet):
+    """
+    Actions to create, update and delete meetings.
+    """
+
+    model = MotionCategory()
+    create_schema = DefaultSchema(MotionCategory()).get_create_schema(
+        properties=["name"], required_properties=["name"],
```

Comment:
You can have a look at OpenSlides 3 to see which fields are required. Maybe the prefix but I am not sure.

Reply:
I think only the name is required. But this is a details that may be fixed later.

Please add the optional properties here that you added to the model.

Reply:
optional properties added